app/models/replication_transfer.rb

```diff
@@ -25,7 +25,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        bag_man_request.cancel!(reason)
+        bag_man_request&.cancel!(reason)
```

Comment:
Never mind, now I see it's new to ruby 2.3.x, i.e.
http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/